examples/cpp/common/Utils.h

```diff
@@ -41,10 +41,6 @@ class RateCounter {
     times.clear();
   }
 
-  unsigned int count() const {
```

Comment:
I don't get why you would remove this.  Knowing the count of measured frames might be valuable to a caller.